bundler/spec/support/artifice/compact_index_wrong_dependencies.rb

```diff
@@ -6,7 +6,7 @@ class CompactIndexWrongDependencies < CompactIndexAPI
   get "/info/:name" do
     etag_response do
       gem = gems.find {|g| g.name == params[:name] }
-      gem.versions.each {|gv| gv.dependencies.clear } if gem
+      gem&.versions&.each {|gv| gv.dependencies.clear }
```

Comment:
RuboCop limits `ChainLength` for this cop to 2. Personally, I think more than one already becomes messy, so I'd go with `ChainLength: 1`.

If others are happy with the default though, then no problem.